Group params together for NW parent templates (NATGW and NAT instances)

dvu70 opened the pull request:
The ones for NAT instances clean up nicely

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/plus3it/cfn/75)
<!-- Reviewable:end -->

lorengordon commented on templates/nw_dualaz_multitier_nat_with_eni.compound:
```diff
@@ -121,6 +121,86 @@
+                {
+                    "Label" :
+                    {
+                        "default": "Private Subnet Configuration"
```
One space both before and after the colon, please. Looks like that applies to all files...

lorengordon commented on templates/nw_tripleaz_multitier_nat_with_eni.compound:
```diff
@@ -141,6 +141,89 @@
+                        "IpAddressNAT2",
+                        "IpAddressNAT3"
+                    ]
+                    
```
Remove the blank line here.

lorengordon commented on templates/nw_singleaz_multitier_nat_with_eni.compound:
```diff
@@ -101,6 +101,83 @@
+                        "KeyPairName",
+                        "IpAddressNAT1"
+                    ]
+                    
```
Remove the blank line

lorengordon commented on templates/nw_dualaz_multitier_nat_with_eni.compound:
```diff
@@ -121,6 +121,86 @@
+                        "IpAddressNAT1",
+                        "IpAddressNAT2"
+                    ]
+                    
```
Blank line...

lorengordon commented:
Just need to fixup the lint mentioned in the line comments.

---

Review status: 0 of 6 files reviewed at latest revision, 4 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/plus3it/cfn/75#-:-KKESFqmUtuNOkEHqqyh:1757128877)*
<!-- Sent from Reviewable.io -->

dvu70 commented:
Fixed formatting

dvu70 commented:
Review status: 0 of 6 files reviewed at latest revision, 4 unresolved discussions.

---

*[templates/nw_dualaz_multitier_nat_with_eni.compound, line 174 \[r1\]](https://reviewable.io:443/reviews/plus3it/cfn/75#-KKENdmSPx3XwviGphUr-r1-174:-KKEjrweqvPlJQ5q824w:610327451) ([raw file](https://github.com/plus3it/cfn/blob/c34d0d704f3d102c516136e9b046e440298181ac/templates/nw_dualaz_multitier_nat_with_eni.compound#L174)):*
<details><summary><i>Previously, lorengordon (Loren Gordon) wrote…</i></summary>
> Blank line...
</details>
Removed

---

*[templates/nw_singleaz_multitier_nat_with_eni.compound, line 151 \[r1\]](https://reviewable.io:443/reviews/plus3it/cfn/75#-KKENdmSPx3XwviGphUt-r1-151:-KKEkeDBNVZEdwy3ar25:610327451) ([raw file](https://github.com/plus3it/cfn/blob/c34d0d704f3d102c516136e9b046e440298181ac/templates/nw_singleaz_multitier_nat_with_eni.compound#L151)):*
<details><summary><i>Previously, lorengordon (Loren Gordon) wrote…</i></summary>
> Remove the blank line
</details>
Removed

---

*[templates/nw_tripleaz_multitier_nat_with_eni.compound, line 197 \[r1\]](https://reviewable.io:443/reviews/plus3it/cfn/75#-KKENdmSPx3XwviGphUv-r1-197:-KKEkgFzvX6_TCFo4xoe:610327451) ([raw file](https://github.com/plus3it/cfn/blob/c34d0d704f3d102c516136e9b046e440298181ac/templates/nw_tripleaz_multitier_nat_with_eni.compound#L197)):*
<details><summary><i>Previously, lorengordon (Loren Gordon) wrote…</i></summary>
> Remove the blank line here.
</details>
Removed

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/plus3it/cfn/75)*
<!-- Sent from Reviewable.io -->

lorengordon commented:
Reviewed 4 of 6 files at r2, 2 of 2 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/plus3it/cfn/75)*
<!-- Sent from Reviewable.io -->
